File: Lab1/t3.cpp
#include <stdio.h>
int main(){
    int start, end;
    printf("Enter the start integer: ");
    scanf("%d", &start);//Scan the first integer
    printf("Enter the end integer: ");
    scanf("%d", &end);//Scan the second integer
    for (int i = start; i <= end; i++){
        printf("%d ", i);//Print out numbers in sequence
    }
    puts("");//Line break
    return 0;
}

File: Lab1/t3.kt
import java.util.*

fun main(args: Array<String>){
    val start: Int
    val end: Int
    val input = Scanner(System.`in`)
    print("Enter the start integer: ")
    start = input.nextInt()
    print("Enter the end integer: ")
    end = input.nextInt()
    for (i in start..end)
        print("$i ")
    println()
}